addons/website_quote/static/src/js/website_quotation.js

```diff
@@ -81,7 +81,18 @@ website.if_dom_contains('div.o_website_quote', function () {
             case "h1":
                 id = _.uniqueId('quote_header_');
                 $(this.parentNode).attr('id',id);
-                sub_li = $("<li>").html('<a href="#'+id+'">'+$(this).text()+'</a>').appendTo(ul);
+                var menu_text = "";
+                var header = $(this).find(".quote-menu");
+                if (header.length){
+                    var node_text = "";
+                    _.each(header,function(node){
+                        node_text = node_text+$(node).text()+" ";
+                    });
+                    menu_text = node_text;
+                }
+                else
+                    menu_text = $(this).text();
+                sub_li = $("<li>").html('<a href="#'+id+'">'+menu_text+'</a>').appendTo(ul);
```

Comment:
@jem-odoo , in this scenario,  under h1 tag some elements text node need to display and some of not. So to identify that quote-menu introduced here.

If we take all children's text than same description appear in menu like "Your Quotation Test/001 Draft Quotation". 

To show only particular description like "Your Quotation Test/001", either we need to change template (for fetching all childs text of h1) or JS code(which is done in this branch).